File: Day24/Solution.cpp
Node* removeDuplicates(Node *head)
          {
            //Write your code here
              Node *current = head;
              while(current->next != NULL){
                  if(current->data == current->next->data){
                     current->next = current->next->next;
                  }
                  else{
                      current = current->next;
                  }
              }

              return head;
          }

File: Day24/Solution.swift
func removeDuplicates(head: Node?) -> Node? {
    var current = head
    while current?.next != nil {
        if current?.data == current?.next?.data {
            current?.next = current?.next?.next
        } else {
            current = current?.next
        }
    }
    return head
} // End of function removeDuplicates
